lib/public/controllers/relatedDevicesCtlr.js

```diff
@@ -15,9 +15,12 @@ angular.module('com.yonomi.testdevice.relatedDevicesCtlr', []).controller('relat
       let _relatedDevice = {
         deviceType: vm.deviceType,
         physicalId: vm.physicalId,
-        deviceId: vm.deviceId
+        deviceId: vm.deviceId,
+        custom: vm.custom
```

Comment:
Why split this up into two lines? Couldn't you just do `custom: JSON.parse(vm.custom)`